internal/admin/dashboard/templates/page-usage.html

```
@@ -153,15 +153,23 @@ <h3>Request Log</h3>
                     <input type="text" placeholder="User path /team/alpha" aria-label="Filter by user path" x-model="usageLogUserPath" @input.debounce.300ms="fetchUsageLog(true)" class="filter-input">
                 </div>
             </div>
+            <div class="usage-filter-row usage-filter-row-options">
+                <label class="usage-log-checkbox">
+                    <input type="checkbox" x-model="usageLogHideCached" @change="fetchUsageLog(true)">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)